src/ucp/wireup/wireup_cm.c

```diff
@@ -1387,23 +1419,12 @@ ucp_request_t* ucp_ep_cm_close_request_get(ucp_ep_h ep, const ucp_request_param_
 
 static int ucp_cm_cbs_remove_filter(const ucs_callbackq_elem_t *elem, void *arg)
 {
-    ucp_cm_client_connect_progress_arg_t *client_connect_arg;
-
-    if (elem->cb == ucp_cm_client_connect_progress) {
-        client_connect_arg = elem->arg;
-        if (client_connect_arg->ucp_ep == arg) {
-            ucp_cm_client_connect_prog_arg_free(client_connect_arg);
-            return 1;
-        } else {
-            return 0;
-        }
-    } else if ((elem->cb == ucp_ep_cm_disconnect_progress)      ||
-               (elem->cb == ucp_cm_server_conn_notify_progress) ||
-               (elem->cb == ucp_cm_client_uct_connect_progress)) {
+    if ((elem->cb == ucp_ep_cm_disconnect_progress) ||
+        (elem->cb == ucp_cm_client_uct_connect_progress)) {
```

Comment:
should also be part of ucp_cm_connect_progress_remove_filter, and rename ucp_cm_cbs_remove_filter to ucp_cm_disconnect_remove_filter

Reply:
done